todo_cli/new.js

```diff
@@ -0,0 +1,25 @@
+const n = (fs, rl, fn) => {
```

Comment:
Consider using a descriptive function name as the function is aliased in `todoCLI.js`